### PR TITLE
Wishlist button added

### PR DESCRIPTION
**Pull request by Smit2010**

### CODE REVIEW

**Comment by Smit2010** [comment]
@pranavpandey1998official Please review...

File: src/components/Property/index.js [hunk 1]
```diff
@@ -33,19 +53,27 @@ class Extended extends Component {
...
+    };
 
 	submitReview = () => {
 		console.log(this.state.reviewInput);
-	};
+    };
+    
+    handleWishlistChange = () => {
+        if(this.state.isWishlisted){
+            this.setState({isWishlisted: false})
```
Comment (pranavpandey1998official) [inline src/components/Property/index.js:72]: we also need to add this to the sever

**Comment by Smit2010** [comment]
@pranavpandey1998official Please review...

File: src/components/Property/index.js [hunk 2]
```diff
@@ -98,12 +176,26 @@ class Extended extends Component {
...
+                        </IconButton>
+                    </BootstrapTooltip>
+                </span>
+            );
+        }
+    };
 
 	render() {
-		const { property } = this.state;
+		const { property, review } = this.state;
```
Comment (pranavpandey1998official) [inline src/components/Property/index.js:196]: this should be `reviews` see your state

File: src/components/Property/index.js [hunk 3]
```diff
@@ -98,12 +176,26 @@ class Extended extends Component {
...
 
-		if (!property) {
+		if (!(property && review)) {
```
Comment (pranavpandey1998official) [inline src/components/Property/index.js:198]: same here

File: package.json [hunk 4]
```diff
@@ -7,6 +7,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.12.1",
     "@fortawesome/react-fontawesome": "^0.1.9",
     "@material-ui/core": "^4.9.5",
+    "@material-ui/icons": "^4.9.1",
```
Comment (pranavpandey1998official) [inline package.json:10]: we definatly don't need this, see how we are using icons in other places

File: src/components/Property/index.js [hunk 5]
```diff
@@ -5,8 +5,34 @@ import { SERVER_URL } from "../../utils/constants";
 import { Carousel } from "react-responsive-carousel";
 import ReactMapGL, { Marker } from "react-map-gl";
 import Review from "./Review";
+import IconButton from '@material-ui/core/IconButton';
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import FavoriteBorderIcon from '@material-ui/icons/FavoriteBorder';
```
Comment (pranavpandey1998official) [inline src/components/Property/index.js:10]: `<i class="far fa-heart"></i>`

File: src/components/Property/index.js [hunk 6]
```diff
@@ -5,8 +5,34 @@ import { SERVER_URL } from "../../utils/constants";
 import { Carousel } from "react-responsive-carousel";
 import ReactMapGL, { Marker } from "react-map-gl";
 import Review from "./Review";
+import IconButton from '@material-ui/core/IconButton';
+import FavoriteIcon from '@material-ui/icons/Favorite';
```
Comment (pranavpandey1998official) [inline src/components/Property/index.js:9]: `<i class="fas fa-heart"></i>`

File: package.json (hunk 4 above)
Comment (Smit2010) [reply package.json:10]: Then toggling of icons is not happening while using font awesome icons.. I don't know why?

Comment (Smit2010) [reply package.json:10]: Then the toggling of the icons is not happening while using <i>. I don't know why?

Comment (pranavpandey1998official) [reply package.json:10]: `RenderWishlistButton = () => {
		const { isWishlisted } = this.state;
        if(isWishlisted){
            return (    
                <span className="icon" key={'filled'}>
                    <BootstrapTooltip title={"Added to Wishlist"}>
                        <IconButton disableRipple onClick={this.handleWishlistChange}>
						<i className={"fas fa-heart" }></i> 
                        </IconButton>
                    </BootstrapTooltip>
                </span>
            );
        } else {
			return (
				<span className="icon" key={'bordered'}>
                    <BootstrapTooltip title={"Added to Wishlist"}>
                        <IconButton disableRipple onClick={this.handleWishlistChange}>
							<i className={"far fa-heart" }></i> 
                        </IconButton>
                    </BootstrapTooltip>
                </span>
			)
		}
    };`

**Pull request merged**